src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -1480,6 +1488,32 @@ public boolean isPaidLessThanClaimAmount() {
     @JsonIgnore
     public boolean isCourtDecisionInClaimantFavourImmediateRePayment() {
         return hasApplicant1CourtDecisionInFavourOfClaimant()
-                && getApplicant1RepaymentOptionForDefendantSpec() == PaymentType.IMMEDIATELY;
+            && getApplicant1RepaymentOptionForDefendantSpec() == PaymentType.IMMEDIATELY;
+    }
+
+    /**
+     * LiP applicant requires Welsh if they selected Welsh or Both languages.
+     *
+     * @return true if applicant is LiP and they accept Welsh (or English+Welsh)
+     */
+    @JsonIgnore
+    public boolean isLipApplicantRequiringWelsh() {
```

Comment:
Is this necessary? There should be a method isClaimantBilingual() and isRespondentResponseBilingual().
Don't they do the same thing?

It might look like they are not as they only look for Language. Both, but that's fine because only both or English is ever sent over from the CUI side (Both/Welsh/English is used for DQs only).